src/Analyzers/CSharp/CodeFixes/FileHeaders/CSharpFileHeaderCodeFixProvider.cs

```diff
@@ -26,8 +26,12 @@ public CSharpFileHeaderCodeFixProvider()
 
         protected override AbstractFileHeaderHelper FileHeaderHelper => CSharpFileHeaderHelper.Instance;
 
-        protected override ISyntaxFacts SyntaxFacts => CSharpSyntaxFacts.Instance;
-
         protected override ISyntaxKinds SyntaxKinds => CSharpSyntaxKinds.Instance;
+
+        protected override SyntaxTrivia EndOfLine(string text)
+            => SyntaxFactory.EndOfLine(text);
+
+        protected override SyntaxTriviaList ParseLeadingTrivia(string text)
+            => SyntaxFactory.ParseLeadingTrivia(text);
```

Comment:
📝 These were internal methods, so I just reverted a6d62cc5d891eaa38900397fc24bacd394868f6b and 1afb1c6c74f691d1bc124e08b8d7fb7aa5bc5f97 to use the original approach.

Reply:
Edit: _Only_ `EndOfLine` was a problem.